mbof/serializers.py

```diff
@@ -18,10 +18,16 @@ class EventSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = Event
         fields = ('url', 'title', 'status', 'address', 'eventText', 'latitude', 'longitude', 'altitudeMeters', 'owner', 'postingTime', 'startTime',
-                  'endTime', 'hashtag', 'votes', 'category',)
+                  'endTime', 'hashtag', 'votes', 'guests', 'category',)
```

Comment:
When you have an intention to go to an event then you are registered as a guest to that event.